plugin/pkg/scheduler/factory/plugins.go

```diff
@@ -99,6 +100,16 @@ func RegisterFitPredicate(name string, predicate algorithm.FitPredicate) string
 	return RegisterFitPredicateFactory(name, func(PluginFactoryArgs) algorithm.FitPredicate { return predicate })
 }
 
+// RegisterRequiredFitPredicate registers a fit predicate with the algorithm registry,
+// it's always included in configuration. Returns the name with which the predicate was registered.
+func RegisterRequiredFitPredicate(name string, predicate algorithm.FitPredicate) string {
```

Comment:
Yes, it does, but we should probably choose a different name for this function and the associated map to make it clear why these predicates are treated differently.